sdk-autoconfigure-support/src/main/java/io/opentelemetry/instrumentation/thread/AddThreadDetailsSpanProcessor.java

```diff
@@ -3,22 +3,29 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.javaagent.tooling;
+package io.opentelemetry.instrumentation.thread;
 
+import static io.opentelemetry.api.common.AttributeKey.longKey;
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+
+import io.opentelemetry.api.common.AttributeKey;
 import io.opentelemetry.context.Context;
 import io.opentelemetry.sdk.common.CompletableResultCode;
 import io.opentelemetry.sdk.trace.ReadWriteSpan;
 import io.opentelemetry.sdk.trace.ReadableSpan;
 import io.opentelemetry.sdk.trace.SpanProcessor;
-import io.opentelemetry.semconv.incubating.ThreadIncubatingAttributes;
 
 public class AddThreadDetailsSpanProcessor implements SpanProcessor {
```

Comment:
should this class be `final` and in an internal package?

Reply:
done